### ISSUE TITLE
Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?

### ISSUE THREAD

**Issue by thqueiroz** [justadudewhohacks/face-api.js#615]
I have some error in withAgeAndGender() node.js

Does anybody can help me?

Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?

**Comment by bdpersons** [comment]
I believe this is an error with the versions. You need to look at the package.json file and see that the @tensorflow/tfjs-node version is 1.7.0. 

Run the command: npm i @tensorflow/tfjs-node@1.7.0 

This should fix the problem, because I had the same problem.

**Comment by thqueiroz** [comment]
> I believe this is an error with the versions. You need to look at the package.json file and see that the @tensorflow/tfjs-node version is 1.7.0.
> 
> Run the command: npm i @tensorflow/tfjs-node@1.7.0
> 
> This should fix the problem, because I had the same problem.

Thank you!! It fixed this problem!
